packages/react-native/src/theme/types.ts

```diff
@@ -40,6 +41,7 @@ export interface Theme {
 
 export interface ComponentStyles {
   // TODO: add components
+  errormessage: ErrorMessageStyles;
```

Comment:
Called this out on another PR but think we should camelCase these names, not sure why the `theme` would follow a casing convention that isn't a javescript idiom